labels.yml

```diff
@@ -16,20 +16,40 @@ labels:
     color: 'f5f5f5'
 
   - name: monitoring
-    description: "This issue should be solved, but it should be monitored for a period before being closed"
+    description: "This issue should be solved, but let's keep an eye on it for a while before we close it."
     color: '357399'
 
   - name: performance
-    description:
+    description: "This issue significantly affects current performance, or will when we scale."
     color: '208116'
 
+  - name: operations
+    description: Anything related to our internal IT systems.
+    color: '262626'
+
+  - name: infrastructure
+    description: Product-related tooling.
+    color: '56a39f'
+
+  - name: debt
+    description: "When we originally built this feature, we knew we'd need to improve it later."
+    color: 'b51e1e'
+
   - name: refactor
-    description:
+    description: "New customer requirements, software updates, or changing business needs are motivating us to improve this."
     color: '627ada'
 
   - name: security
-    description:
-    color: '3b4047'
+    description: A significant customer-facing security issue.
+    color: '191a47'
+
+  - name: dev_research
+    description: "We know the outcome. How might we solve it?"
+    color: '9083bd'
+
+  - name: client_services
```

Comment:
The `client_services` label would apply to all client projects.

Two things:

- How would you distinguish `enterprise` from `ops`?
- `enterprise` could be interpreted to include [enterprise features](https://github.com/dobtco/screendoor-v2/milestone/85) that SaaS customers might also use , as well as deployment-related tasks. That doesn't seem like your intent.